navigation/dead_reckoning/sam_dead_reckoning/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>sam_dead_reckoning</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
```

Comment:
"This package contains the nodes required to consume IMU/etc data to produce dead reckoning for the SAM AUV"